src/components/GeneralInfo/InfoTable/InfoTable.js

```diff
@@ -15,172 +15,173 @@ import {
   TableHeader,
 } from '@patternfly/react-table/deprecated';
 import flatMap from 'lodash/flatMap';
-import {
-  filterRows,
-  generateFilters,
-  onDeleteFilter,
-  prepareRows,
-} from '../../../constants';
-class InfoTable extends Component {
-  state = {
-    sortBy: { index: 0, direction: SortByDirection.asc },
-    opened: [],
-    pagination: {
-      page: 1,
-      perPage: 10,
-    },
-    activeFilters: {},
-  };
+import { filterRows, generateFilters, prepareRows } from '../../../constants';
+
+const InfoTable = (props) => {
+  const {
+    cells = [],
+    rows = [],
+    expandable = false,
+    filters,
+    onSort = undefined,
+  } = props;
```

Comment:
This is not wrong, but usually the assignment can already be in the arguments assigning like:

```js
const InfoTable = ({
    cells = [],
    rows = [],
    expandable = false,
    filters,
    onSort = undefined,
  }) => {
```

Just having the `prop` argument and using it for assignments would only be useful if we were to further pass on the same exact props to a child component.